ci/testnet-manager.sh

```diff
@@ -487,7 +487,7 @@ deploy() {
       fi
 
       if [[ -z $INTERNAL_NODES_LAMPORTS ]]; then
-        maybeInternalNodesLamports="--internal-nodes-lamports 2000000000" # 2 SOL
+        maybeInternalNodesLamports="--internal-nodes-lamports 500000000000" # 500 SOL
```

Comment:
@mvines this line is the TdS config (default overrides anyway), just wanted to make that won't be any issue for that specific cluster.

Reply:
Should we just remove `INTERNAL_NODES_LAMPORTS`?  We don't use define it anymore in the TdS setup env: https://buildkite.com/solana-labs/tds-create-and-start/settings

Reply:
Are you proposing removing the code block https://github.com/solana-labs/solana/pull/7411/files/ad636d5f8896d189ff21ab1c4c59c2911a0f4e93#diff-30a451266d7e4b951f83d31a44881d5dL489-L493 ?

This will force us to use the `net/` default values.  I think there might be value in not removing the knob for this deployment.  We don't define that env var `INTERNAL_NODES_LAMPORTS` in the pipeline, because we have a working value right here in the code, but we still could use it if we wanted to.

Reply:
Yeah I'm proposing we remove the dead code.

![image](https://user-images.githubusercontent.com/1281082/70576945-66e3ad80-1b67-11ea-9364-b086afcde5e2.png)